app/src/main/java/com/mobile/tandil/javabaseproject/mvp/presenter/ParkingFragmentPresenter.java

```diff
@@ -0,0 +1,20 @@
+package com.mobile.tandil.javabaseproject.mvp.presenter;
+
+import com.mobile.tandil.javabaseproject.listener.ListenerDialogFragment;
+import com.mobile.tandil.javabaseproject.mvp.contract.ParkingFragmentContract;
+
+public class ParkingFragmentPresenter implements ParkingFragmentContract.Presenter {
+
+    private final ParkingFragmentContract.View view;
+
+    public ParkingFragmentPresenter(ParkingFragmentContract.View view) {
+        this.view = view;
+    }
+
+    @Override
+    public void onPressAcceptedButton(String number, ListenerDialogFragment listener) {
+        if(!number.isEmpty()){
```

Comment:
Add and space after 'if' and other space between crackt and curly

Reply:
done